### PR TITLE
docs: removing first person from 2 sentences

### PR DESCRIPTION
**Pull request by maksymiliansielicki**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

n/a

## What is the new behavior?
n/a

## Does this PR introduce a breaking change?

- [ ] Yes
- [x ] No

### CODE REVIEW

**Comment by maxmeister** [comment]
@kapunahelewong Hi! I've made the changes you suggested in https://github.com/angular/angular/pull/36188 :)

File: aio/content/guide/dependency-injection-providers.md [hunk 1]
```diff
@@ -197,11 +197,11 @@ Like  `EvenBetterLogger`, `HeroService` needs to know if the user is authorized
 That authorization can change during the course of a single application session,
 as when you log in a different user.
 
-Let's say you don't want to inject `UserService` directly into `HeroService`, because you don't want to complicate that service with security-sensitive information.
+Imagine that you don't want to inject `UserService` directly into `HeroService`, because you don't want to complicate that service with security-sensitive information.
 `HeroService` won't have direct access to the user information to decide
 who is authorized and who isn't.
 
-To resolve this, we give the `HeroService` constructor a boolean flag to control display of secret heroes.
+To resolve this, the `HeroService` constructor is given a boolean flag to control display of secret heroes.
```
Comment (kapunahelewong) [inline aio/content/guide/dependency-injection-providers.md:204]: How about this? It keeps the sentence in the active voice:
```suggestion
To resolve this, give the `HeroService` constructor a boolean flag to control display of secret heroes.
```

**Review by kapunahelewong — COMMENTED** [review]
Yay! Thank you, @maxmeister! Just one tiny suggestion about the changes. Then to get it to pass, we just need to meet the 100 character minimum for the body of the commit message. If you try this, it should pass:

```
docs: remove first person from 2 sentences 

This commit removes two instances of the first person in the 
Dependency injection providers documentation.

```
Make sure you have a blank line before and after the body of the commit message just like this.^^

If you need any help, just let me know. You may also be interested in the notes I keep on how to [squash](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-squash) (which you'll need to do if you have a second commit) and how to [amend a commit](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-amend-a-commit).

**Review by AndrewKushnir — APPROVED** [review]

**Comment by maxmeister** [comment]
I've made the changes you suggested. Thank you so much for help! I use Angular in my daily work and contributing to it means a lot to me

**Comment by kapunahelewong** [comment]
How nice to hear, @maxmeister. 😊

And it is so nice that you've taken the time to help make Angular better. Thank you! 🌸 

LGTM!

Reviewed-for: global-docs-approvers

**Review by kapunahelewong — APPROVED** [review]

**Pull request closed without merging**